snapshot/src/test/java/io/zeebe/snapshots/broker/impl/ReceivedSnapshotTest.java

```diff
@@ -193,33 +200,33 @@ public void shouldReturnAlreadyExistingSnapshotOnPersist() throws Exception {
     final var transientSnapshot = senderSnapshotStore.newTransientSnapshot(index, term, 1, 0).get();
     transientSnapshot.take(
         p -> takeSnapshot(p, List.of("file3", "file1", "file2"), List.of("content", "this", "is")));
-    final var persistedSnapshot = transientSnapshot.persist();
+    final var persistedSnapshot = transientSnapshot.persist().join();
     final var firstReceivedSnapshot =
         receiverSnapshotStore.newReceivedSnapshot(persistedSnapshot.getId());
     try (final var snapshotChunkReader = persistedSnapshot.newChunkReader()) {
       while (snapshotChunkReader.hasNext()) {
         firstReceivedSnapshot.apply(snapshotChunkReader.next());
       }
     }
-    final var alreadyPeristedSnapshot = firstReceivedSnapshot.persist();
+    final var alreadyPeristedSnapshot = firstReceivedSnapshot.persist().join();
 
     // when - receives same snapshot again
     final var receivedSnapshot =
         receiverSnapshotStore.newReceivedSnapshot(persistedSnapshot.getId());
     try (final var snapshotChunkReader = persistedSnapshot.newChunkReader()) {
       while (snapshotChunkReader.hasNext()) {
-        final var success = receivedSnapshot.apply(snapshotChunkReader.next());
+        final var success = receivedSnapshot.apply(snapshotChunkReader.next()).join();
         assertThat(success).isTrue();
       }
     }
-    final var persistedReceivedSnapshot = receivedSnapshot.persist();
+    final var persistedReceivedSnapshot = receivedSnapshot.persist().join();
 
     // then
     assertThat(persistedReceivedSnapshot).isEqualTo(alreadyPeristedSnapshot);
     assertThat(persistedReceivedSnapshot == alreadyPeristedSnapshot).isTrue();
   }
 
-  @Test
+  /*@Test
```

Comment:
This test is commented out and, judging by the TransientSnapshot interface changes, should be removed

Reply:
:see_no_evil: